spec/stub_helper.rb

```diff
@@ -0,0 +1,134 @@
+def cross_streets
```

Comment:
i don't know where you're using this (maybe i missed something), but i think this is an AWESOME idea 🎉